tests/components/devolo_home_network/test_sensor.py

```diff
@@ -1,15 +1,17 @@
 """Tests for the devolo Home Network sensors."""
+from datetime import timedelta
```

Comment:
Can you move the sensor changes to a separate PR?
It's a chunkier change than for the other platforms.

Reply:
Sure, I will seperate that.